### PR TITLE
chore : remove django-tailwind

### PR DESCRIPTION
**Pull request by cheikhgwane**
Remove django-tailwind. (See https://github.com/BLSQ/openhexa-team/issues/598) for reference.

## Changes

Please list / describe the changes in the codebase for the reviewer(s).

- Add generated css file to static folder.
- Delete static_src folder.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -24,14 +24,6 @@ CMD start
 
 FROM deps as app
```
Comment (pvanliefland) [inline Dockerfile:25]: It seems that this whole build stage can be removed - could you try? We should probably rename deps to app.

**Review by pvanliefland — CHANGES_REQUESTED** [review]
@cheikhgwane nice work, could you just check whether we can get rid of one of the Dockerfile build step.

File: Dockerfile (hunk 1 above)
Comment (cheikhgwane) [reply Dockerfile]: Okay.

**Review by pvanliefland — APPROVED** [review]
LGTM, let's test it!

**Pull request merged**